db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,102 @@
+class AddCommentsToVmsTable < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines and Templates"
+
+    change_column_comment :vms, :autostart, "Indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "ID of the availability zone that the VM is part of."
+    change_column_comment :vms, :busy, "Indicates whether or not the VM is being scanned for SmartState analysis. Deprecated."
+    change_column_comment :vms, :boot_time, "The last time the VM was started."
+    change_column_comment :vms, :cloud, "Indicates whether or not the VM/Template is cloud or infra."
+    change_column_comment :vms, :cloud_network_id, "ID of the cloud (virtual) network the VM/Template is associated with."
+    change_column_comment :vms, :cloud_subnet_id, "Subnet ID associated with the VM."
+    change_column_comment :vms, :cloud_tenant_id, "ID of the cloud tenant associated with the VM."
+    change_column_comment :vms, :config_xml, "The VM or Template configuration in XML format. Deprecated."
+    change_column_comment :vms, :connection_state, "Indicates the connection state of the VM, e.g. connected, disconnected, etc."
+    change_column_comment :vms, :cpu_affinity, "A comma separated list of host CPUs to schedule the VM on. VMware only."
+    change_column_comment :vms, :cpu_hot_add_enabled, "Indicates if CPUs can be added to the VM while it is powered on."
+    change_column_comment :vms, :cpu_hot_remove_enabled, "Indicates if a CPU can be removed from a VM while powered on."
+    change_column_comment :vms, :cpu_limit, "The CPU utilization limit for the VM. VMWare only."
+    change_column_comment :vms, :cpu_reserve, "Amount of CPU that is guaranteed available for the VM. VMWare only."
+
+    change_column_comment :vms, :cpu_reserve_expand, "Amount the CPU can grow beyond its normal limits if there "\
+                                                     "are unreserved resources available. VMWare only."
+
+    change_column_comment :vms, :cpu_shares, "The number of CPU shares allocated. Used to determine resource allocation "\
+                                             "in case of resource contention. VMWare only."
+
+    change_column_comment :vms, :cpu_shares_level, "CPU shares allocation level. VMWare only."
+    change_column_comment :vms, :created_on, "The timestamp the VM was added to the app inventory."
+    change_column_comment :vms, :deprecated, "Indicates whether or not a template is deprecated. Google only."
+    change_column_comment :vms, :description, "A description of the VM that is typically set by the provider, but may also be edited."
+    change_column_comment :vms, :ems_created_on, "Date the associated EMS was created."
```

Comment:
Perhaps "Date the VM or Templated was created on the EMS itself".  The current text sounds like it's when the EMS itself was created

Reply:
Or perhaps start with "The timestamp", since you've used that wording for other datetime columnes